suitcase/spec.py

```diff
@@ -965,11 +965,13 @@ def to_spec_file_header(start, filepath, baseline_descriptor=None):
 
 _SPEC_1D_COMMAND_TEMPLATE = env.from_string("{{ plan_name }} {{ scan_motor }} {{ start }} {{ stop }} {{ num }} {{ time }}")
 
-_SCANS_WITHOUT_MOTORS = ['ct']
-_SCANS_WITH_MOTORS = ['ascan', 'dscan']
-_SPEC_SCAN_NAMES = _SCANS_WITH_MOTORS + _SCANS_WITHOUT_MOTORS
+_SCANS_WITHOUT_MOTORS = {'ct': 'count'}
```

Comment:
I see that this section has been refactored from collections to dicts. What's the reasoning behind this -- what is being mapped to what? I think we have count-like plans `['ct', 'count']` and scan-like plans `['ascan', 'scan', 'dscan', 'rel_scan']` and we need to recognize which group (if any) a given run represents. I don't think we need to _normalize_ the plan name -- we just need to classify as count-like or scan-like so that we know what columns to put in our table.

Reply:
Dict provides an easy way to map between spec name and bs plan name. Agree, we can figure out better groups for spec style scan as more names are added.